### ISSUE TITLE
Monitoring State of Backups

### ISSUE THREAD

**Issue by christheradioguy** [Sebclem/hassio-nextcloud-backup#223]
Is it possible to add a sensor to report the state of the backups?  I'd love to be able to use an automation to send a notification if the last successful backup was over 24 hours ago or something similar.

**Comment by kdickerson** [comment]
Yes, please! I just discovered that no backups were being sent to Nextcloud for 15 months....because I went to grab one to restore a crashed system.  I was....not happy.

Reporting whether backups are being successfully generated and transferred is _critical_ functionality that I had no idea was not implemented.

**Comment by RHCPNG** [comment]
Love to have something like this too! +1
